### DISCUSSION TITLE
Ability to lurk in any public channels without joining it

### DISCUSSION THREAD

**Discussion by darkLord19** [mattermost/community#142] [General Ideas]
Sometimes I just want to look what channel is about but to do so I have to join that channel and then everyone gets notified that X joined channel and then I will leave immediately if it is not relevant to me and that will create even more channel noise

**Comment by esethna** [comment]
Related in a way: https://mattermost.atlassian.net/browse/MM-24495

Does that solve you primary use case?

**Reply by darkLord19** [reply]
Yup this is exactly what I had in mind
